internal/api/verify.go

```diff
@@ -432,18 +454,18 @@ func (a *API) emailChangeVerify(r *http.Request, ctx context.Context, conn *stor
 	return user, nil
 }
 
-func (a *API) verifyEmailLink(ctx context.Context, conn *storage.Connection, params *VerifyParams, aud string, flowType models.FlowType) (*models.User, error) {
```

Comment:
@J0 seems like we don't use the `flowType` param at all in this function so i removed it